Buildings/Resources/Scripts/BuildingsPy/conf.yml

```diff
@@ -171,12 +171,14 @@
   optimica:
     comment: timeout on travis after 300s, https://app.travis-ci.com/github/lbl-srg/modelica-buildings/jobs/618700275
     time_out: 600
+- model_name: Buildings.Fluid.CHPs.OrganicRankine.Validation.VariableSource
```

Comment:
See above comment regarding how to correct the division by zero. I think we will get this model to work with OpenModelica.

Reply:
Fixed in 9501acd and 38abd36.